0016-intern-like-system.md

```diff
@@ -0,0 +1,53 @@
+# Intern Like system
+
+* Status: proposed 
+<!-- {proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)} --> <!-- optional -->
+* Input: Welcome from anyone!
+* Deciders: {list everyone involved in the decision using github handle}
+<!-- Find deciders here: https://github.com/orgs/Greenstand/people  -->
+* Date: 2022-09-09 <!-- optional -->
+
+
+## Context and Problem Statement
+
+The Facebook Like button on the web map currently has some challenges with the Eurpean Region, https://developers.facebook.com/docs/plugins/like-button
+
+Another issue is that we are not able to create a custom Like button in the Greenstand style, https://github.com/Greenstand/treetracker-web-map-client/pull/395
+
+## Decision Drivers <!-- optional -->
+
+* {driver 1, e.g., a force, facing concern, …}
+
+## Considered Options
+
+* Option #1 - Intern Like system, to save likes on trees/entities directly
+* Option #2 - Your suggestions are welcome
+
+## Decision Outcome
+
+<!-- Chosen option: "{option 1}", because {justification. e.g., only option, which meets k.o. criterion decision driver | which resolves force {force} | … | comes out best (see below)}. -->
+
+To be decided.
+
+### Positive Consequences <!-- optional -->
+
+Option #1
+* Users do not need an social media account, for example Facebook
+* The like data becomes our own
+* Implementation of a custom UI Like button would be possible, currently we have a standard Facebook Like button
+
+### Negative Consequences <!-- optional -->
+
+Option #1
+* Risk of "cheating", not being able to identify the user
+
```

Comment:
If we don't link "likes" to a user account, the user experience may be confusing. A user returning to a previously "liked" tree may see no record that they liked it, causing them to do so again.

Reply:
I'm also worried about this, but there might be some trade-off way to identify the user, by collecting some fingerprint based on browser agent or so? or use cookie, locale storage